easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb

```diff
@@ -53,9 +53,13 @@ exts_list = [
         'checksums': ['f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98'],
     }),
     ('numpy', '1.15.0', {
+        'patches': ['numpy-1.12.0-mkl.patch'],
```

Comment:
@akesandgren This doesn't belong here, it's a patch for numpy with MKL, which is not relevant when the `foss` toolchain is being used...

Reply:
Yeah, I figured that out going back from lunch. Wasn't paying enough attention at the time.